docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.SYNC_TRANSFER_START 0
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.AXI_SLICE_SRC 0
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.AXI_SLICE_DEST 1
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.DMA_2D_TRANSFER 0
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.DMA_DATA_WIDTH_SRC 32
+   ad_ip_parameter axi_pulsar_adc_dma CONFIG.DMA_DATA_WIDTH _DEST 64
+   ad_connect spi_clk axi_pulsar_adc_dma/s_axis_aclk
+
+**Debug options – ILA**
+
+.. figure:: use_case_debug_options_ila.png
+   :align: center
+
+**Debug options – regular SPI controller transfer capture**
+
+.. figure:: use_case_debug_options_spi.png
+   :align: center
+
+**Debug options –SPI Engine controller transfer capture**
+
+.. figure:: use_case_debug_options_spie.png
+   :align: center
+
+Build System
+~~~~~~~~~~~~
+
+.. figure:: system_build_bd.png
+   :align: center
+
+**System Build - ADALM2000**
+
+.. figure:: system_build_m2k.png
+   :align: center
+
+- Two programmable power supplies
+- Two-channel USB digital oscilloscope
+- Two-channel arbitrary function generator
+- 16-channel digital logic analyzer (3.3V CMOS and 1.8V or 5V tolerant, 100MS/s)
+
+**System Build - Scopy**
+
+.. figure:: system_build_scopy.png
+   :align: center
+
+Uses the ADALM2000 to implement virtual instruments:
+- Oscilloscope (with Mixed Signal Capability)
+- Signal Generator (Functions and Arbitrary)
+- Spectrum Analyzer
+- Network Analyzer
+- Voltmeter
+- Power Supply
+- Logic Analyzer (with Stack Decoder support)
+- Digital Pattern Generator
+- GPIO
+
+**System Build - Schematic**
+
+.. figure:: system_build_schematic.png
+   :align: center
+
+**System Build - CORA Z7S Configuration**
+
+.. figure:: system_build_cora.png
+   :align: center
+
+**System Build - Power Supply**
+
+.. figure:: system_build_power_supply.png
+   :align: center
+
+**System Build - Input Signal**
+
+.. figure:: system_build_input_signal.png
+   :align: center
+
+**System Build - UART Configuration**
+
+.. figure:: system_build_uart.png
+   :align: center
+
+**System Build – change host IP**
+
+.. figure:: system_build_ip.png
+   :align: center
+
+**System Build - UART and Ethernet Testing**
+
+.. figure:: system_build_uart_and_eth.png
+   :align: center
+
+Evaluate System
+~~~~~~~~~~~~~~~
+
+**System Evaluation – regular SPI trigger configuration**
+
+.. figure:: system_evaluation_spi_trigger_1.png
+   :align: center
+
+.. figure:: system_evaluation_spi_trigger_2.png
+   :align: center
+
+**System Evaluation**
+
+=========================================================== ====================== =====================
+Steps                                                       Regular SPI controller SPI Engine controller
+=========================================================== ====================== =====================
+Connect IIO Oscilloscope and visualize the captured signals
+Run the Python script on Cora z7s / Cygwin
+Analyze results in VisualAnalog
+Compare the results
```

Comment:
Is this correct? no checkbox is checked

Reply:
is correct, this is the first step - at this level we didn't do any step for now. If you look down, in the next table steps begin to be marked as completed